lib/src/main/java/com/thenewboston/data/dto/bankapi/invalidblockdto/request/PostInvalidBlockRequest.kt

```diff
@@ -0,0 +1,63 @@
+package com.thenewboston.data.dto.bankapi.invalidblockdto.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PostInvalidBlockRequest(
+
+    @SerialName("message")
+    val message: PostInvalidBlockMessage,
+
+    @SerialName("node_identifier")
+    val nodeIdentifier: String,
+
+    @SerialName("signature")
+    val signature: String,
+)
+
+@Serializable
+data class PostInvalidBlockMessage(
+
+    @SerialName("block")
+    val block: PostInvalidBlockMessageBlock,
```

Comment:
Rename it to `Block` Keep the names to what the API calls it so it doesn't become confusing.